### PR TITLE
Box-list fixes

### PR DESCRIPTION
**Pull request by MariCerv**
- new button fixed
- container nabber fixed
- background color ( boxes and page ) fixed
- font size box-extra-info fixed
- modal button “create” fixed

### CODE REVIEW

**Comment by pjaneiro** [comment]
I've checked with @MariCerv and it's working, but she still wants to change the way new boxes are highlighted, don't merge it yet

File: boxes/templates/boxes/box_list.html
Comment (dethos) [inline boxes/templates/boxes/box_list.html]: The indentation does not seem to be correct here, and bellow.

File: boxes/templates/boxes/box_list.html
Comment (dethos) [inline boxes/templates/boxes/box_list.html]: We are trying to avoid inline javascript. Please move the code to `javascripts/box_create.js` or add this classes to the elements, since this seems to run every time.

**Review by dethos — CHANGES_REQUESTED** [review]
Overall it seems good, but the indentation is not correct due to a subtle issue on one file and we can't allow inline JS. As soon as it is fixed I will merge.

File: boxes/templates/boxes/box_list.html
Comment (pjaneiro) [reply boxes/templates/boxes/box_list.html]: It's not easy to add the classes to the elements since they're created dynamically by Django...

Comment (dethos) [reply boxes/templates/boxes/box_list.html]: @Budgerous ok, so moving to the external file seems to be the right solution.

**Review by dethos — APPROVED** [review]

**Pull request merged**
